Ktisis/Structs/Bones/CustomOffset.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Numerics;
+
+using Ktisis.Structs.Actor;
+
+namespace Ktisis.Structs.Bones {
+	public class CustomOffset {
+		public unsafe static Vector3 GetBoneOffset(Bone bone) {
+			var target = Ktisis.Target; // TODO: Get bone's owner actor instead of target's
+
+			if (!Ktisis.Configuration.CustomBoneOffset.TryGetValue(GetRaceGenderFromActor(target), out var bonesOffsets))
+				return new();
+			if (!bonesOffsets.TryGetValue(bone.HkaBone.Name.String, out Vector3 offset))
+				return new();
+			return offset;
+		}
+
+		// changed from BodyType to string Race_Gender
+		public unsafe static string GetRaceGenderFromActor(Actor.Actor* actor) =>
+			$"{actor->Customize.Race}_{actor->Customize.Gender}";
+		public unsafe static BodyType GetBodyTypeFromActor(Actor.Actor* actor) {
+
+			var gender = actor->Customize.Gender;
+			var race = actor->Customize.Race;
+
+			if (gender == Gender.Male)
+				switch (race) {
+					case Race.Lalafell: return BodyType.LalafellMale;
+					case Race.Roegadyn:
+					case Race.Hrothgar: return BodyType.HrothgarRoegadyn;
+					default: return BodyType.TallMale;
+				} else switch (race) {
```

Comment:
This is not correctly indented.
At minimum, I'd suggest this:
```suggestion
				} 
			else 
				switch (race) {
```

But ideally, add braces around this. It's a multi-line statement, we can afford those.

Reply:
VS offers this, i'm not sure if it's worth.
```suggestion
			else return race switch {
				Race.Lalafell => BodyType.LalafellFemale,
				_ => BodyType.TallFemale,
			};
```

Reply:
I would still break the line there. Could do both?

Reply:
Yes I noticed it was unrelated with the else being on the wrong indent (fixed with 1acc264).
I was curious of your opinion about "switch statement expression" style offered by vs.

Reply:
Switch expressions are neat, imo. Don't see a reason not to use them.
(Though if you apply it to the `else` branch, you should probably apply it to the `if` branch as well, for consistency)